crates/llm/Cargo.toml

```diff
@@ -33,7 +33,7 @@ default = ["models", "tokenizers-remote"]
 
 tokenizers-remote = ["llm-base/tokenizers-remote"]
 
-models = ["llama", "gpt2", "gptj", "bloom", "gptneox", "mpt"]
+models = ["llama", "gpt2", "gptj", "bloom", "gptneox", "mpt", "falcon"]
```

Comment:
We should still keep falcon behind a feature flag, until the problem with the linearly scaling performance of the repeat operation is fixed.

Reply:
I agree with your opinion.